triggertask/spring-cloud-starter-stream-source-triggertask/README.adoc

```diff
@@ -0,0 +1,24 @@
+//tag::ref-doc[]
+= TriggerTask Source
+
+The TriggerTask app sends a TaskLaunchRequest based on a fixed delay, date or cron expression.
+
+== Options
+The **$$triggertask$$** $$source$$ has the following options:
+
+$$commandLineArgs$$:: $$space separated key=value pairs to be used as commandline arguments for the task.$$ *($$String$$, default: ``)*
+$$properties$$:: $$comma separated key=value pairs to be used as properties for the task.$$ *($$String$$, default: ``)*
```

Comment:
We should document what the difference between command line args and properties should be used for.